disttask/framework/storage/task_table.go

```diff
@@ -78,15 +79,8 @@ func execSQL(ctx context.Context, se sessionctx.Context, sql string, args ...int
 		return nil, err
 	}
 	if rs != nil {
-		rows, err := sqlexec.DrainRecordSet(ctx, rs, 1)
-		if err != nil {
-			return nil, err
-		}
-		err = rs.Close()
-		if err != nil {
-			return nil, err
-		}
-		return rows, err
+		defer terror.Call(rs.Close)
+		return sqlexec.DrainRecordSet(ctx, rs, 1024)
```

Comment:
Why change `maxChunkSize` to 1024? I think most internal SQLs should not return so many rows.

Reply:
It may get many row when getting all pending subtasks.